app/views/food/show.html.erb

```diff
@@ -0,0 +1,9 @@
+
+
+
+ <% @music_stuff.each do |thing| %>
+ <%= thing.name %>
+ <%= thing.type %>
+ <br>
+
+ <% end %>
```

Comment:
I noticed extra white space and missing indentation in a few places, definitely try to tidy that up.